Workflow rule: run only on GitHub pull requests

heerener opened the pull request.

heerener committed:
Workflow rule: run only on GitHub pull requests

olupton reviewed (commented):
We could also follow https://github.com/neuronsimulator/nrn/blob/36fde9ec44f48b72fb5838dffe59524bb9a43c1e/.gitlab-ci.yml#L5 and use https://bbpgitlab.epfl.ch/hpc/gitlab-pipelines/-/blob/main/github-project-pipelines.gitlab-ci.yml.

heerener committed:
Use hpc/gitlab-pipelines instead of defining our own rules

heerener committed:
$GITLAB_PIPELINES_BRANCH is not needed

Originally copied from neuron, which is a pipeline triggered by
hpc/gitlab-pipelines CI to check helper modifications.
Anything not in this case can simply use the default branch.

heerener committed:
Show CI_PIPELINE_SOURCE; there may be duplicate pipelines

heerener committed:
Revert "Show CI_PIPELINE_SOURCE; there may be duplicate pipelines"

This reverts commit 0972b5eb42c313bdc34709ced866c5afbe26ab12.

heerener committed:
Revert "Revert "Show CI_PIPELINE_SOURCE; there may be duplicate pipelines""

This reverts commit f10a4357e8c9c75ec3a0e38e12ae7783e38152e0.

heerener committed:
Kick pipeline

heerener committed:
Remove debug info again
